restapi.go

```diff
@@ -1454,6 +1454,76 @@ func (s *Session) GuildEmojiDelete(guildID, emojiID string, options ...RequestOp
 	return
 }
 
+// ApplicationEmojis returns all emojis for the given application
+// appID : ID of the application
+func (s *Session) ApplicationEmojis(appID string, options ...RequestOption) (emoji []*Emoji, err error) {
```

Comment:
Since this func returns multiple emojis, I think it's better to name return param to `emojis`.